server/services/v1/service.go

```diff
@@ -35,11 +36,12 @@ type Service interface {
 	RegisterGRPC(grpc *grpc.Server) error
 }
 
-func GetRegisteredServices(kvStore kv.KeyValueStore, searchStore search.Store, tenantMgr *metadata.TenantManager, txMgr *transaction.Manager) []Service {
+func GetRegisteredServices(kvStore kv.KeyValueStore, searchStore search.Store, tenantMgr *metadata.TenantManager, txMgr *transaction.Manager, defaultConfig config.Config) []Service {
 	var v1Services []Service
 
 	v1Services = append(v1Services, newApiService(kvStore, searchStore, tenantMgr, txMgr))
 	v1Services = append(v1Services, newHealthService())
 	v1Services = append(v1Services, newAdminService(tenantMgr, txMgr))
+	v1Services = append(v1Services, newAuthService(defaultConfig.Auth))
```

Comment:
can we just access config.DefaultConfig here and not pass it around?

Reply:
updated.